doc/combining.rst

```diff
@@ -237,3 +238,64 @@ coordinates as long as any non-missing values agree or are disjoint:
 Note that due to the underlying representation of missing values as floating
 point numbers (``NaN``), variable data type is not always preserved when merging
 in this manner.
+
+.. _combining.multi:
```

Comment:
Can you add a warning or note box talking about `auto_combine` vs `combine_auto`? I think that's going to be confusing, so the docs should be clear.